### ISSUE TITLE
HW 2 Feedback

### ISSUE THREAD

**Issue by alexchaomander** [bingbingboo/DAT_SF_12#4]
- Nice use of plots to help you determine the number of neighbors for KNN
- Good use of scale. Adding the additional scaling jumps the predictive accuracy from 70% to 90%. Exploration of your data through plots and other means would have revealed this.
- There are other ways to calculate the accuracy of your model. For instance most models will have a score function. i.e. "clf.score(X_test, y_test)" http://scikit-learn.org/stable/modules/generated/sklearn.cluster.KMeans.html
- And while the problem didn't specify it, I think using a few basic histograms to visualize the distribution of the data can go a long way in catching things like the differing scales and other peculiarities with which you can modify your model.  
- Also add comments or a short writeup comparing the models you've done. This will not only help the reader but also yourself when you go back to look at your code. 
- All in all, good work!
